templates/records/record_page.html

```diff
@@ -1,8 +1,13 @@
 {% extends "base.html" %}
 
 {% load static %}
+{% load i18n %}
 {% load datalayer_tags %}
 
+{% block title %}
+    {{ page.title }} - {% trans 'The National Archives' %}
```

Comment:
Thanks James, good fix!

Wagtail passes uses two keys for the current page, `page` and `self`:

https://github.com/wagtail/wagtail/blob/7bf3f813efe8f25e11ddf68a20fd1165f5cd67da/wagtail/core/models/__init__.py#L1092-L1102

The reason why the page title isn't output in `base.html` is because the view used to fetch the record data from Kong is only using `page`.

https://github.com/nationalarchives/ds-wagtail/blob/3f52a64245efd4ed99ce0039d77e60cc760710e8/etna/records/views/records.py#L66-L67

`base.html` uses `self`, which is inconsistent with the rest of our templates. A quick fix for this would be to rename `self` to `page` in `base.html` and then all pages will populate `<title>` in the same way.

Reply:
I must give credit to @gtvj for that one! Thanks @danbentley, I've made those changes but "The National Archives" doesn't seem to be coming through for the details page title and I'm not too sure why

Reply:
I must give credit to @gtvj for that one. Thanks @danbentley, I've made those changes but I can't figure out how to get the "The National Archives" suffix to display on details pages

Reply:
Ah, apologies James, I've only given you half a solution there!

When rendering a page. Wagtail queries which site the page belongs to using `Page.get_site` (a single Wagtail instance can host multiple sites)

https://github.com/wagtail/wagtail/blob/7bf3f813efe8f25e11ddf68a20fd1165f5cd67da/wagtail/core/models/__init__.py#L1278

Since the `RecordPage` isn't CMSable it technically doesn't belong to any site. 

But we can override the `get_site` method to return the appropriate site instance:

```# in records/models.py
from wagtail.core.models import Site

class RecordPage(Page):
   …
   def get_site(self):
         """Override to return the Site instance despite this page not belonging to the CMS."""
         return Site.objects.first()
   …
```

I'm happy to make this change if you'd like.

Reply:
Thanks Dan, that would be great if you could. Apologies, I know it's a simple change but I'm getting errors when I try running Wagtail locally atm

Reply:
No problem, I've pushed a commit.

Reply:
Thanks Dan :)